src/templates/en/2019/table_of_contents.html

```diff
@@ -156,6 +156,29 @@
 <section class="main">
   <h1 class="title">Table of Contents</h1>
 
+  <section>
+    <h2>Foreword</h2>
+
+    <p>
+      The open web is an amazingly complex, evolving network of technologies. Entire industries and careers are built on the web and depend on its vibrant ecosystem to succeed. As critical as the web is, understanding how it's doing has been surprisingly elusive. Since 2010, the mission of the <a href="https://httparchive.org/">HTTP Archive</a> project has been to track how the web is built. And it's been doing an amazing job of it. But there has been one gap that has been especially challenging to close: bringing meaning to the data and enabling the web community to easily understand how it's performing. That's where the Web Almanac comes in.
+    </p>
+
+    <p>
+      The mission of the Web Almanac is to take the treasure trove of insights that would otherwise be accessible only to intrepid data miners and package it up in a way that's easy to understand. This is made possible with the help of industry experts who can make sense of the data and tell us what it means. Each of the 20 chapters in the Web Almanac focuses on a specific aspect of the web and it's authored and peer reviewed by experts in their field. The strength of the Web Almanac flows directly from the expertise of the people who write it.
```

Comment:
```suggestion
      The mission of the Web Almanac is to take the treasure trove of insights that would otherwise be accessible only to intrepid data miners and package it up in a way that's easy to understand. This is made possible with the help of industry experts who can make sense of the data and tell us what it means. Each of the 20 chapters in the Web Almanac focuses on a specific aspect of the web and it's authored and peer reviewed by experts in their field. The strength of the Web Almanac flows directly from the expertise of the people who write it and all the other contributors who have made this project possible. {{ contributors }} people have volunteered their time for this project. 
```

Reply:
Yeah this is tough. I don't mean to undervalue others' contributions, it's just that the authors are the ones speaking directly to the readers and communicating the insights. We highlight all of our contributors in many other places on the site so I think it's ok to focus on the contents here.